package.json

```diff
@@ -20,8 +20,9 @@
     "async": "0.9.0",
     "autosize": "^1.18.13",
     "bootstrap": "3.1.1",
-    "i18next": "3.4.4",
-    "i18next-xhr-backend": "1.1.0",
+    "i18next": "7.0.0",
+    "i18next-browser-languagedetector": "*",
```

Comment:
Hum i18n is not possible cause it clashes with variable in Translate Component inside react-i18next.